traffic_ops/app/lib/API/Cdn.pm

```diff
@@ -264,7 +264,8 @@ sub get_traffic_monitor_config {
 		{ 'cdn.name' => $cdn_name },
 		{   prefetch => ['cdn', 'profile', 'type'],
 			select   => 'me.profile',
-			distinct => 1
+			distinct => 1,
```

Comment:
I know you didn't add it, just commenting. My patch would involve removing DBIx. @mitchell852 knows what I am referring to. We went through a handful of slow queries that were all slow because `DISTINCT` was added.